### PR TITLE
 Certificate verify error is thrown when syncing S3

### PR DESCRIPTION
**Pull request by akanshi-crest**
#### Pre-Review Checklist
- [x] Covered the changes with automated tests
- [x] Tested the changes locally
- [x] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [x] Considered corresponding documentation changes
- [x] Contributed any configuration settings changes to the configuration reference

### CODE REVIEW

File: docs/DEVELOPING.md [hunk 1]
```diff
@@ -276,3 +276,7 @@ To implement a new source, check [CONTRIBUTE.rst](./CONTRIBUTING.md)
 
 - MacOS or Linux server. The connector has been tested with CentOS 7, MacOS Monterey v12.0.1.
 - Python version 3.10 or later.
+- To fix SSL certificate verification failed error, users have to run this to connect with Amazon S3:
```
Comment (tarekziade) [inline docs/DEVELOPING.md:279]: I would add a section about installing Python 3.10, and for macOS this extra step, independently from the S3 connector.

Comment (akanshi-crest) [reply docs/DEVELOPING.md:279]: So if you are planning to add a section regarding the installation, do you want me to close this PR ?

Comment (tarekziade) [reply docs/DEVELOPING.md:279]: no that's fine as a first step, thanks!

**Review by tarekziade — APPROVED** [review]

**Comment by tarekziade** [comment]
buildkite test this

**Pull request merged**